ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/ImageVulnerabilitiesTable.tsx

```diff
@@ -218,11 +219,22 @@ function ImageVulnerabilitiesTable({
                                 <Td />
                                 <Td colSpan={colSpan}>
                                     <ExpandableRowContent>
-                                        <p className="pf-u-mb-md">{summary}</p>
-                                        <ImageComponentVulnerabilitiesTable
-                                            imageMetadataContext={image}
-                                            componentVulnerabilities={imageComponents}
-                                        />
+                                        {summary && image ? (
+                                            <>
+                                                <p className="pf-u-mb-md">{summary}</p>
+                                                <ImageComponentVulnerabilitiesTable
+                                                    imageMetadataContext={image}
+                                                    componentVulnerabilities={imageComponents}
+                                                />
+                                            </>
+                                        ) : (
+                                            <Alert
+                                                isInline
+                                                variant="warning"
+                                                title="Partial CVE data"
+                                                aria-label="Partial CVE data"
+                                            />
+                                        )}
```

Comment:
It looks like we have this logic reused in 3 different locations. We check for a condition (the common condition being that a summary needs to exist). If it does, we render some component(s). If it doesn't, we show the partial data alert. Is it worthwhile to create something reusable here in case we decide to make future changes?

Reply:
Hmm that is a good point. I don't think we would want to make a component that accepts a boolean and a `children` prop to handle the conditional rendering, since that seems too complex for this simple use case. It might be worthwhile abstracting the `<Alert>` into a `<PartialCveDataAlert>` though, to avoid the repetition of the props in each case.